src/Jackett.Common/Services/UpdateService.cs

```diff
@@ -332,26 +311,25 @@ private async Task<string> DownloadRelease(List<Asset> assets, bool isWindows, s
 
         private void StartUpdate(string updaterExePath, string installLocation, bool isWindows, bool NoRestart, bool trayIsRunning)
         {
-            string appType = "Console";
+            var appType = "Console";
 
-            if (isWindows && windowsService.ServiceExists() && windowsService.ServiceRunning())
-            {
-                appType = "WindowsService";
-            }
+            if (isWindows && windowsService.ServiceExists() && windowsService.ServiceRunning()) appType = "WindowsService";
 
             var exe = Path.GetFileName(ExePath());
-            var args = string.Join(" ", Environment.GetCommandLineArgs().Skip(1).Select(a => a.Contains(" ") ? "\"" + a + "\"" : a )).Replace("\"", "\\\"");
+            var args = string.Join(" ", Environment.GetCommandLineArgs().Skip(1).Select(a => a.Contains(" ") ? $"\"{a}\""
+                : a )).Replace("\"", "\\\"");
 
-            var startInfo = new ProcessStartInfo();
-            startInfo.UseShellExecute = false;
-            startInfo.CreateNoWindow = true;
+            var startInfo = new ProcessStartInfo
+            {
+                UseShellExecute = false,
+                CreateNoWindow = true
+            };
 
             // Note: add a leading space to the --Args argument to avoid parsing as arguments
             if (variant == Variants.JackettVariant.Mono)
             {
                 // Wrap mono
-                args = exe + " " + args;
-                exe = "mono";
```

Comment:
args is used, but it's switched to string interpolation `$"{exe} {args}"` instead of adding together. The extra line of changing `exe` to mono is unnecessary though, because exe isn't used again.